app/src/main/java/com/google/droidjump/GameView.java

```diff
@@ -141,6 +148,9 @@ private void drawScene() {
         if (surfaceHolder.getSurface().isValid()) {
             Canvas canvas = getHolder().lockCanvas();
             canvas.drawColor(Color.WHITE);
+            String levelHeader = GAME_LEVEL_HEADER + " " + currentLevel;
```

Comment:
use   String.format to construct this string.

Reply:
Used.